### DISCUSSION TITLE
How to use document driven with i18n?

### DISCUSSION THREAD

**Discussion by oripka** [nuxt/content#1741] [General]
It seems document driven mode is used by a lot of nuxt documentation sites. What I didn’t see is an i18n in document driven mode.

I saw it should be possible if I write the queries myself. But that’s exactly the point of document driven mode, that I can concentrate on writing Markdown and not layouts, components, navigation and queries.

Any hints in this direction?

**Comment by Barbapapazes** [comment]
As I understand, Docus will soon support I18n. I think we can expect an update with this!
